app/assets/javascripts/admin/addon/components/schema-theme-setting/types/group.gjs

```diff
@@ -0,0 +1,32 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { action } from "@ember/object";
+import Group from "discourse/models/group";
+import GroupChooser from "select-kit/components/group-chooser";
+
+export default class SchemaThemeSettingTypeGroup extends Component {
+  @tracked value;
+  @tracked groups;
+
+  constructor() {
+    super(...arguments);
+    this.value = this.args.value;
+    Group.findAll().then((groups) => {
+      this.groups = groups;
+    });
+  }
+
+  @action
+  onInput(newVal) {
+    this.value = newVal;
+    this.args.onChange(newVal);
+  }
+
+  <template>
+    <GroupChooser
```

Comment:
This seems to be allowing multiple groups to be chosen but we don't support that on the backend yet. Only 1 group id field